pkg/registry/resource/resourceclaimtemplate/strategy.go

```diff
@@ -113,10 +113,46 @@ func toSelectableFields(template *resource.ResourceClaimTemplate) fields.Set {
 
 func dropDisabledFields(newClaimTemplate, oldClaimTemplate *resource.ResourceClaimTemplate) {
```

Comment:
Should we dedupe the logic between this and the ResourceClaim strategy? I think it just needs a `func dropDisabledFields(newResourceClaimSpce, oldResourceClaimSpec *resource.ResourceClaimSpec)`?